code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -244,7 +244,12 @@
 		/obj/item/device/flashlight,
 		/obj/item/weapon/cell/device,
 		/obj/item/weapon/extinguisher/mini,
-		/obj/item/weapon/storage/quickdraw/syringe_case
+		/obj/item/weapon/storage/quickdraw/syringe_case,
```

Comment:
Odd, this is actually a duplicate and should probably be deleted.

Reply:
Good catch